components/tidb_query/src/batch/executors/util/mod.rs

```diff
@@ -40,9 +39,13 @@ pub unsafe fn eval_exprs_decoded_no_lifetime<'a>(
     input_logical_rows: &[usize],
     output: &mut Vec<RpnStackNode<'a>>,
 ) -> Result<()> {
+    unsafe fn erase_lifetime<'a, T: ?Sized>(v: &T) -> &'a T {
```

Comment:
Is it better to add `#[inline]` here?

Reply:
I'm 99% sure that rustc will inline this